### PR TITLE
Adding additional checks for HashMap_T.java

### PR DESCRIPTION
**Pull request by sandredd**
Adding assertNotNull after put in testClear Unit Test

### CODE REVIEW

**Comment by sandredd** [comment]
Three test cases run successfully without implementing put. Adding checks to make sure put is implemented before proceeding.

File: java/test/edu/berkeley/cs/util/HashMap_T.java [hunk 1]
```diff
@@ -20,6 +20,7 @@ public void testClear() {
 
     Assert.assertEquals(1, map.size());
     Assert.assertFalse(map.isEmpty());
+    Assert.assertNotNull(map.get(1));
```
Comment (fsareshwala) [inline java/test/edu/berkeley/cs/util/HashMap_T.java:23]: This function is testing `clear` so I don't want to start introducing other checks here like testing that `get` works properly

File: java/test/edu/berkeley/cs/util/HashMap_T.java [hunk 2]
```diff
@@ -108,6 +109,8 @@ public void testDelete() {
       map.put(keys[i], random.nextInt());
     }
 
+    Assert.assertNotNull(map.get(1));
```
Comment (fsareshwala) [inline java/test/edu/berkeley/cs/util/HashMap_T.java:112]: This function is testing `delete` so I don't want to start introducing other checks here like testing that get works properly

File: java/test/edu/berkeley/cs/util/HashMap_T.java [hunk 3]
```diff
@@ -128,6 +131,8 @@ public void testIterator() {
       map.put(key, value);
     }
 
+    Assert.assertNotNull(map.get(1));
```
Comment (fsareshwala) [inline java/test/edu/berkeley/cs/util/HashMap_T.java:134]: This function is testing the iterator so I don't want to start introducing other checks here like testing that get works properly

**Pull request closed without merging**